lms/templates/student_account/login_and_register.html

```diff
@@ -1,6 +1,7 @@
 <%!
```

Comment:
@douglashall Thanks! Do you have any idea why my devstack doesn't catch these issues when I run `paver run_quality` (here's the full output: https://pastebin.com/QTsv5yFy)? Maybe there's some caching going on? I tried running `paver update_assets lms --settings=devstack && paver update_assets cms --settings=devstack` first.